build/config/compiler/BUILD.gn

```diff
@@ -301,6 +301,18 @@ config("compiler") {
         "_LARGEFILE64_SOURCE",
       ]
     }
+  } else if (is_win) {
+    if (is_clang) {
+      if (current_cpu == "x86") {
```

Comment:
Does x86 actually work?

Reply:
No clue; all the support has been there since before we forked. My guess based on this experience and also reviving the stale Windows clang code is that in terms of just our build system, it's probably more or less working, and the problems would be all the expected issues with Dart.

We could theoretically rip it all out, but I don't want to mix that with this PR. (And since we have an open-even-if-unlikely-to-be-addressed issue requesting x86 Windows support, I'm not sure we actually want to go out of our way to do that.)